src/components/books/books.css

```diff
@@ -0,0 +1,127 @@
+.booklistContainer {
+  margin-top: 3rem;
+}
+
+.bookContainer {
+  display: grid;
+  grid-template-columns: 50% 25% 15%;
+  background: #fffcfc;
+  list-style: none;
+  margin-bottom: 1.2rem;
+  margin-left: 5%;
+  margin-right: 5%;
+  padding: 1rem;
+  box-shadow: 20px 20px 30px 20px #e6e1e1;
+}
+
+.bookContent > small {
+  color: #999;
+  font-weight: 800;
+}
+
+.bookContent > h2 {
+  margin: 2px;
+}
+
+.bookContent p {
+  margin-top: 2px;
+  color: blue;
+}
+
+.callContainer {
+  display: flex;
+  gap: 1rem;
+}
+
+.callContainer > button {
+  padding: 0.6rem;
+  border: none;
+  color: blue;
+}
+
+.progressContainer {
+  display: grid;
+  grid-template-columns: 20% 2%;
+  gap: 2rem;
+  position: relative;
+  align-items: center;
+}
+
+.circle {
+  width: 4.25rem;
+  height: 4.25rem;
+  border-radius: 50%;
+  border: 4px solid rgb(51, 182, 226);
+  border-width: 5px;
+}
+
+.percentage h1 {
+  font-size: 3rem;
+  font-weight: 200;
+  margin: 0;
+}
+
+.percentage p {
+  margin: 0;
+  color: rgb(139, 136, 136);
+}
+
+.progressContainer::after {
+  content: "";
+  position: absolute;
+  right: 15%;
+  width: 2px;
+  height: 4rem;
+  background: rgb(223, 216, 216);
+}
+
+.currentChapter {
+  color: #999;
+  margin-bottom: 0;
+}
+
+.updateContainer p {
+  margin-top: 2px;
+  font-size: 1.1rem;
+}
+
+.progressCtn {
+  background-color: rgb(51, 182, 226);
+  color: #fff;
+  font-weight: 600;
+  border: none;
+  padding: 0.8rem;
+  margin-top: 1.5rem;
+  border-radius: 0.2rem;
+}
+
+.input {
+  margin: 2rem;
+  padding: 2rem;
+}
+
+.input > h1 {
+  color: #999;
+}
+
+form {
+  display: grid;
+  grid-template-columns: 1fr 0.5fr 0.2fr;
+  gap: 1rem;
+  width: 80%;
+}
+
+input {
+  padding: 0.8rem;
+}
```

Comment:
- [ ] Kindly make sure your input field looks as much as possible to the provided design.

   - ### YOUR DESIGN
   
       ![image](https://user-images.githubusercontent.com/87197412/216769585-091caae7-511b-4c48-9568-cf31cb3a453f.png)

   - ### PROVIDED DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/216769606-1f095d26-4720-4ef2-927a-67821a22eda2.png)